CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# How To Contribute
+
+## Some Ways to Contribute
+
+- Report potential bugs
+- Suggest product enhancements
+- Increase our test coverage
+- Fix a bug
+- Implement a requested enhancement
+- Improve our guides and documentation
+- Contribute code
+
+## Clone and Provision Environment
+
+1. Make sure you have a GitHub account
+2. Fork the l3afd repository to your GitHub user or organization. 
+3. Clone your ${YOUR_GITHUB_USERNAME_OR_ORG}/l3afd fork into your GOPATH, and setup the base repository as upstream remote:
+
+```
+mkdir -p "${GOPATH}/src/github.com/l3af-project"
+cd "${GOPATH}/src/github.com/l3af-project"
+git clone https://github.com/${YOUR_GITHUB_USERNAME_OR_ORG}/l3afd.git
+cd l3afd
+git remote add upstream https://github.com/l3af-project/l3afd.git
+```
+4. Setup your [L3AF Development Environment on vagrant](https://github.com/l3af-project/l3af-arch/tree/main/dev_environment). 
```

Comment:
```suggestion
4. Set up your [L3AF Development Environment on vagrant](https://github.com/l3af-project/l3af-arch/tree/main/dev_environment). 
```
Is Vagrant really required for all contributions?  For example, if I'm contributing changes specific to Windows, why is that step required?